src/main/java/org/codehaus/mojo/versions/DisplayDependencyUpdatesMojo.java

```diff
@@ -257,6 +258,94 @@
     @Parameter( property = "verbose", defaultValue = "false" )
     private boolean verbose;
 
+    /**
+     * Only take these artifacts into consideration.
+     * <p>
+     * Comma-separated list of extended GAV patterns.
+     *
+     * <p>
+     * Extended GAV: groupId:artifactId:version:type:classifier:scope
```

Comment:
I'm not sure if scope is used in plugin -> dependencies - IMHO it is always runtime - to check or ask on dev list